cli/normalize_us_address.py

```diff
@@ -0,0 +1,26 @@
+import sys
+import os
+
+sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), ".."))
+
+from external.googleapi import normalize_us_address  # noqa: E402
+
+usage = """
+usage:
+    python normalize_us_address.py "6 Jay St"
+"""
+
+
+def main():
```

Comment:
could we add some comments describing what/why is going on here? Just cause I know I'll look at this in like 3 weeks and have no idea what is going on lol.

Reply:
Let me add some comments.
How about adding a sample output and a short description to the usage above?
When you run `python normalize_us_address.py` without arguments, it will prints usage like:

```
usage:
    python normalize_us_address.py "6 Jay St"

```

And, the modified one would be:
```
Normalize an address to the standard form. It normalizes upper/lowercase letter and a partial address, etc.

usage:
    python normalize_us_address.py "6 jay st"

output:
    6 Jay St Brooklyn  NY USA

```

Reply:
sounds perfect